src/provider.ts

```diff
@@ -128,6 +132,11 @@ export class DelphinusBrowserConnector extends DelphinusProvider<BrowserProvider
     }
   }
 
+  async addAndSwitchNet(network: AddNetworkOptions) {
```

Comment:
What we want to do is "try add if not exist and then switch"
So is this addNetwork do the "try add“ ? If the metamask wallet had the same chain id network added (maybe name, urls are different), it will use it correctly and not add duplicate one?

Reply:
Tested it will not add duplicate chainid network so it is ok